client/modules/wj-css/src/base/_typography.scss

```diff
@@ -30,6 +39,14 @@
                       monospace,
                       var(--font-system-default);
 
+
+  // --font-sans is effectively the default font.
+  // --font-display is for headings and the like.
+  // --font-serif is just a good ol' serif font.
+  // --font-mono is meant for code or console output text.
+
+  // TODO: figure out what fonts we actually want to use
+
   --font-sans:    "Public Sans",   var(--font-system-sans);
   --font-display: "Exo 2",         var(--font-system-rounded);
   --font-serif:   "Bitter",        var(--font-system-serif);
```

Comment:
Btw, when we get closer to frontend integration, how will our font choices here interact with Sigma-9, Black Highlighter, or other custom themes that sites use? How comparable are these font choices to whatever Wikidot's defaults are?

Reply:
In regards to theming, I don't think font choice is too big a deal. As long as the font has enough features (like what weights it has, italics, etc.), it should be fine.

The font choices here are fairly well thought out imo but they were for Mainframe, not for Wikijump. So there's a bit of a theme going on with them - e.g. Public Sans is literally a U.S. government made font, and I thought that added a certain bit of character for the choice considering it was specifically for the SCP wiki (I still think the font works for Wikijump regardless).

Reply:
I see. Copying Mainframe fonts for now makes sense, but with regards to Wikijump default fonts, we'll need a discussion in the branding channel where we look at Wikidot's default fonts and compare them and decide if we're making them official.

Reply:
Yeah there is a few factors to consider, such as:
* Variable fonts (all fonts here are variables except Sometype Mono)
* How these fonts handle non latin alphabets
* What the system font stack should be (I still think my list is up to date mostly)
* What extra fonts we may want to make available, such as a print handwriting font and a cursive font (Mainframe has these but I didn't bring them over)
* Potentially look into providing some simple clientside configuration for fonts such as enabling dyslexia fonts

I haven't actually seen that last one done before, but I know that those fonts really help people and it seems like such a simple thing that I want to try it at some point.